packages/applications-service-api/src/services/submission.service.js

```diff
@@ -66,7 +67,22 @@ const updateSubmission = async (submission) => {
 	});
 };
 
+const getSubmission = (submissionId) => db.Submission.findOne({ where: { id: submissionId } });
+
+const completeSubmission = async (submissionId) => {
+	const submission = await getSubmission(submissionId);
+
+	if (!submission) {
+		throw ApiError.notFound(`Submission with ID ${submissionId} not found`);
+	}
+
+	console.log(`Found submission with ID ${submission.id}, email: ${submission.email}`);
```

Comment:
I don't think we should log the email as it's personal information.

Reply:
That's fair enough, I added this line just temporarily so the endpoint does "something" and was going to remove it in the next story. It's not really needed so I will remove now